x/inflation/keeper/msg_server.go

```diff
@@ -0,0 +1,26 @@
+package keeper
+
+import (
+	"context"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/evmos/evmos/v10/x/inflation/types"
+	"github.com/pkg/errors"
```

Comment:
Is this import intended? or you want to use the `Wrapf()` from the `errorsmod "cosmossdk.io/errors"` in all cases?
```suggestion
```